scripts/vars.sh

```diff
@@ -220,6 +220,7 @@ GAIA_REDEMPTION_ICA_ADDR='cosmos1xmcwu75s8v7s54k79390wc5gwtgkeqhvzegpj0nm2tdwacv
 GAIA_WITHDRAWAL_ICA_ADDR='cosmos1x5p8er7e2ne8l54tx33l560l8djuyapny55pksctuguzdc00dj7saqcw2l'
 GAIA_FEE_ICA_ADDR='cosmos1lkgt5sfshn9shm7hd7chtytkq4mvwvswgmyl0hkacd4rmusu9wwq60cezx'
 GAIA_RECEIVER_ACCT='cosmos1g6qdx6kdhpf000afvvpte7hp0vnpzapuyxp8uf'
+GAIA_REVENUE_ACCT=''
```

Comment:
nit: why not rm?

Reply:
removed in latest commit - I wasn't sure what this was used for at the time but I later realized it was just an alias of the fee ICA and removed it